src/lib/transition_frontier/persistent_root/persistent_root.ml

```diff
@@ -107,7 +107,7 @@ module Instance = struct
   let potential_snarked_ledgers_to_yojson queue =
     `List
       (List.map (Queue.to_list queue) ~f:(fun config ->
-           `String (Ledger.Root.Config.primary_directory config) ) )
+           Ledger.Root.Config.to_yojson config ) )
```

Comment:
If this is changed to `Config.to_yojson` then the `potential_snarked_ledgers_of_yojson` needs to change too.

Reply:
Make sense. Will fix on this.